src/connectors/argent/argentMobile/modal/argentModal.ts

```diff
@@ -25,6 +30,7 @@ const overlayStyle = {
   fontFamily: "'Barlow', sans-serif",
 }
 
+// TODO - SK-47 - remove this
```

Comment:
checked SK-47 and saw it's the new UI
this means this part will be also moved in its own package (that will just be used internally by starknetkit)?